src/Microsoft.AspNetCore.Sockets.Client.Http/HttpConnection.cs

```diff
@@ -404,9 +403,9 @@ private void SetTransferMode(TransferMode transferMode)
             }
         }
 
-        public async Task DisposeAsync() => await DisposeAsyncCore().ForceAsync();
+        public async Task DisposeAsync(CancellationToken cancellationToken = default) => await DisposeAsyncCore(cancellationToken).ForceAsync();
```

Comment:
What is the point of taking cancellation token here if it is not used? Is it for custom implementations?

Reply:
That's something I wanted to ask about. It's listed in the redesign notes but I'm unsure about the actual usage..